nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -111,6 +111,30 @@ let
     }
   ''));
 
+  accessLogString = { file, format, buffer, gzip, flush, off ? [], ... }:
+    (if off then "
+      access_log off;"
+    else "
+      access_log "
+        + optionalString (file != null) "${file} "
+        + optionalString (buffer != null || gzip != null || (buffer != null && flush != null)) "${format} "
+        + optionalString (buffer != null) "buffer=${buffer} "
+        + optionalString (gzip == 1) "gzip "
+        + optionalString (gzip != 1 && gzip != null) "gzip=${toString gzip} "
```

Comment:
We can't set gzip=1?

Reply:
This way of recording is shown in the examples in the nginx documentation. I can return `gzip=1` variant.